src/Services/Room/index.js

```diff
@@ -57,7 +57,7 @@ const joinRoom = async ({
 
   room.users.push({
     id: generateId(),
-    username,
+    name: username,
```

Comment:
```suggestion
    username,
```
Tüm projenin geli kalan kısmında `username` ifadesi kullanıcldığından dolayı kafa karışıklığı oluşturmamamk için burayı da bence `username` olarak tercih etmeliyiz